README.md

```diff
@@ -70,6 +70,27 @@ import { localize } from "nativescript-localize";
 console.log(localize("Hello world !"));
 ```
 
+### Vue
+#### main.js
+
+You have to manually install the global filter
+
+```js
+import { localize } from "nativescript-localize";
+
+Vue.filter('L', localize)
+```
+
+#### Template
+```xml
+<Label :text="'Hello world !'|L"></Label>
+<Label :text="'I am %s'|L('user name')"></Label>
+```
+
+#### Developing
+
+Beware that, if you are using *vue-cli-template*, the localization files placed under *dist/app/i18n* must be manually cleaned since they don't are correctly updated each time you modify them.
```

Comment:
Could you try to automate this by updating you webpack configuration ? I could add that in the FAQ, but I think steps to follow should be clearer.

Reply:
I suppose I could but I'm not a webpack expert and right now it's not my priority since the localisation files don't change so often. But if in the future I'll do it, I'll come back with a detailed solution. For the moment I think that what I wrote is still better than nothing, no?
There is just one step to follow: run 

     rm dist/app/i18n/*

If you want I can be more specific in the readme.